models/ngo.model.js

```diff
@@ -24,6 +24,7 @@ const NGOSchema = new mongoose.Schema(
         item_count: { type: Number, default: 0 },
       },
     ],
+    description: { type: String, required: true }
```

Comment:
It should not be a required field, I guess

Reply:
Okay, as of now we'll not keep it as a required field.